config

```diff
@@ -27,3 +25,4 @@ export RH_CHE_DOCKER_IMAGE_VERSION=${RH_CHE_DOCKER_IMAGE_VERSION:-${DEFAULT_RH_C
 export DOCKER_HUB_USER=${DOCKER_HUB_USER:-${RHCHEBOT_DOCKER_HUB_USER}}
 export DOCKER_HUB_PASSWORD=${DOCKER_HUB_PASSWORD:-${RHCHEBOT_DOCKER_HUB_PASSWORD}}
 export DOCKER_HUB_NAMESPACE=${DOCKER_HUB_NAMESPACE:-${RH_CHE_DOCKER_HUB_NAMESPACE}}
+export DOCKER_HUB_REGISTRY_PREFIX=${DOCKER_HUB_REGISTRY_PREFIX:-""}
```

Comment:
well, there's also the following line:
```
    docker login -u ${DOCKER_HUB_USER} -p $DOCKER_HUB_PASSWORD -e noreply@redhat.com  ${DOCKER_HUB_REGISTRY_PREFIX}
```
and for this line the solution you propose won't work afaict.

However I assume this line should be moved to the condition executed when it's a CI build (non-developer build).
In the developer use-cases (`dev_buildXXX` scripts), we don't push docker images to any remote repository, so we don't need to login afaik.